### DISCUSSION TITLE
Colourblind mode

### DISCUSSION THREAD

**Discussion by OLF19** [f1multiviewer/issue-tracker#276] [Feature requests] [answered]
For me as a colourblind person it's really hard to notice the difference between the yellow and the green sector times and mini sectors in the live timing. Maybe a colourblind mode with colours which have more contrast could be added as an option.

**Comment by f1multiviewer** [comment, marked as answer]
Hey there! This is actually something that's available! Others have asked this too. If you go to Settings, there's an Accessibility section that allows you to customize almost any color used in the app, including colors for flags, sectors, drivers etc., which was built in collaboration with other users with color deficiency. Does this help for you too?

I always try to use icons for things as well (instead of just a color) to make sure they are more recognizable. Let me know if I missed something 💚

**Reply by OLF19** [reply]
thank you, I will try it
